tests/test_security_system.py

```diff
@@ -11,14 +15,18 @@ def test_access_logging(self):
         # Ensure access attempts are logged correctly
         person_name = 'Test User'
         is_authorized = True
-
-        # Call the log_access_attempt method
-        self.security_system.log_access_attempt(person_name, is_authorized)
-
-        # Check if the access_logs.log file contains the log entry
-        with open('app/access_logs.log', 'r') as log_file:
-            log_contents = log_file.read()
-            self.assertIn(f'Person: {person_name}, Authorized: {is_authorized}', log_contents)
```

Comment:
This test is supposed to check if the logs are correctly logged. I think we can also check the log format.

Reply:
I think `mocked_logger.assert_called_with(f'Person: {person_name}, Authorized: {is_authorized}')` is already checking the format.